examples/src/migration/README.rst

```diff
@@ -0,0 +1,22 @@
+##################
+Migration Examples
+##################
+
+The `Encryption SDK for Python`_ now uses the `AWS Cryptographic Material Providers Library`_. The MPL abstracts lower
+level cryptographic materials management of encryption and decryption materials.
+
+This directory contains migration examples for:
+
+* Moving to Keyrings from Master Key Providers:
+    #. Migration example to AWS KMS keyring from AWS KMS Master Key Provider.
+    #. Migration example to Raw AES keyring from Raw AES Master Key Provider.
+    #. Migration example to Raw RSA keyring from Raw RSA Master Key Provider.
+    
+* Migration to newer versions of the ESDK from the old version (1.x):
+    #. Setting a 'CommitmentPolicy' during migration.
```

Comment:
I think this might be slightly misformatted? The rich text doesn't look quite right

Reply:
Yeah the line break after line 16 was causing the issue. Changed this to keep everything there in a single line. Thanks.